.github/workflows/pkgdown.yaml

```diff
@@ -35,6 +35,7 @@ jobs:
             nflverse/nflplotR
             any::tidyverse
             any::ggrepel
+            any::ggimage
```

Comment:
also needs nflverse/nflreadr to make the pkgdown go

Reply:
added, although I have no idea why the pkgdown action now fails and in previous commit it did not